llvm/lib/Target/AArch64/SVEInstrFormats.td

```diff
@@ -10349,6 +10349,16 @@ multiclass sve2_luti2_vector_index<string mnemonic> {
     let Inst{23-22} = idx{2-1};
     let Inst{12}    = idx{0};
   }
+
```

Comment:
I can see lots of white spaces  when I push the patch at the end of:
def : SVE_3_Op_Imm_Pat
and
def : Pat<(nxv8i16 (int_aarch64_sve_luti4_lane_x2

Reply:
I am not sure I understood this, but I removed extra whitespaces

Reply:
Done